src/App.tsx

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const prepareQuery = query.trim().toLowerCase();
+
+    return movie.title.toLowerCase().includes(prepareQuery)
+    || movie.description.toLowerCase().includes(prepareQuery);
+  });
```

Comment:
Let's create variables to improve readability

```suggestion
  const visibleMovies = moviesFromServer.filter(movie => {
    const preparedQuery = query.trim().toLowerCase();
    const preparedTitle = movie.title.toLowerCase();
    const preparedDesc = movie.description.toLowerCase();

    return preparedTitle.includes(prepareQuery)
    || preparedDesc.includes(prepareQuery);
  });
```